advanced_guides/binary.md

```diff
@@ -0,0 +1,70 @@
+# MeiliSearch Binary
+
+
+## How to compile meiliSearch
+
+### Dependencies 
+
+MeiliSearch is made in `rust`. Because of that `rust` and his package manager `cargo` must [be installed](https://www.rust-lang.org/tools/install) to compile the project.
+
+### Compiling
+
+Once this is done, first we need to clone the project and go to the cloned directory. 
```

Comment:
Maybe precise what is this "this" in "Once this is done" ? For example, the "Once the rust installation"

Reply:
"Once rust and cargo are installed, we need to clone the project and go to the cloned directory. "

Reply:
Now that you have Rust and Cargo installed on your machine, we need to download and move into the MeiliSearch project.